recipe-client-addon/test/browser/browser_PreferenceExperiments.js

```diff
@@ -397,6 +401,7 @@ add_task(withMockExperiments(withMockPreferences(async function(experiments) {
 // stop should remove a preference that had no value prior to an experiment for user prefs
 add_task(withMockExperiments(withMockPreferences(async function(experiments, mockPreferences) {
   const stopObserver = sinon.stub(PreferenceExperiments, "stopObserver");
+  mockPreferences.set("fake.preference", "defaultvalue", "default");
```

Comment:
This shouldn't be necessary, the test asserts using `isSet` now, which only checks user preferences.